packages/ecs-gpu-init/ecs-gpu-init.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Initialize ECS agent GPUs config
+Documentation=https://aws.amazon.com/documentation/ecs/
```

Comment:
This is probably not the best documentation URL; I'd either omit it or point it into our own repo.